kolibri/core/assets/src/core-app/constructor.js

```diff
@@ -8,6 +8,13 @@ const vuex = require('vuex');
 const Mediator = require('./mediator');
 const ResourceManager = require('../api-resource').ResourceManager;
 const Resources = require('../api-resources');
+const rest = require('rest');
+const mime = require('rest/interceptor/mime');
+const csrf = require('rest/interceptor/csrf');
+const errorCode = require('rest/interceptor/errorCode');
+const cookiejs = require('js-cookie');
```

Comment:
Yes, I have left third party libraries requireable in this way, mostly to hoover up any lax importing.